cmd/palomad/root.go

```diff
@@ -93,7 +100,7 @@ func NewRootCmd() *cobra.Command {
 		flags.FlagKeyringBackend: keyring.BackendOS,
 	}); err != nil {
 		// seems to be non-fatal issue
-		log.Printf("error overriding flag defaults: %s", err.Error())
+		fmt.Println("error overriding flag defaults: %s", err.Error())
```

Comment:
Why not use the log package?

Reply:
We imported `"cosmossdk.io/log"`  which does not have Printf function so its updated to fmt.

Reply:
pushed the changes with "log" module

Reply:
We don't encourage using the `fmt()` package for logging. You can use `log.Info()` instead.